### PR TITLE
Add audit message for list members request

### PR DESCRIPTION
**Pull request by yoavgeva**
### Desired Outcome

This PR adds an audit log message for List members request

### Implemented Changes

*Describe how the desired outcome above has been achieved with this PR. In
particular, consider:*

- Example audit message

**Successful Audit:**
```
<86>1 2022-12-29T12:11:02.467Z - conjur 22348 members [auth@43868 user="cucumber:user:admin"][subject@43868 role="cucumber:group:dev"][client@43868 ip="127.0.0.1"][action@43868 result="success" operation="list"] cucumber:user:admin successfully listed members with parameters: {:account=>"cucumber", :kind=>"group"}
```
**Failed Audit:**
```
<84>1 2022-12-29T12:11:07.732Z - conjur 29411 members [auth@43868 user="cucumber:user:admin"][subject@43868 role="cucumber:group:dev"][client@43868 ip="127.0.0.1"][action@43868 result="failure" operation="list"] cucumber:user:admin failed to list members with parameters: {:account=>"cucumber", :kind=>"group", :limit=>"abc"}: Limits must be greater than or equal to 1
```

### Connected Issue/Story

Resolves [ONYX-26095](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-26095)

CyberArk internal issue ID: [ONYX-26095](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-26095)

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [x] This PR includes new unit and integration tests to go with the code
  changes, or
- [ ] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]
- [x] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [x] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [x] There are no security aspects to these changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - OIDC Authenticator can now be configured to distribute access tokens with a
   custom time-to-live.
   [cyberark/conjur#2683](https://github.com/cyberark/conjur/pull/2683)
+- List members request (`GET /members`) now produce audit events.
```
Comment (aloncarmel111) [inline CHANGELOG.md:35]: please move it to the unreleased version

File: app/controllers/roles_controller.rb [hunk 2]
```diff
@@ -212,8 +217,11 @@ def filtered_roles(roles, filter)
   end
 
   def render_dataset(dataset, &block)
-    resp = count_only?  ? count_payload(dataset) :
-           block_given? ? yield(dataset)         : dataset.all
+    resp = if count_only?
```
Comment (aloncarmel111) [inline app/controllers/roles_controller.rb:220]: why?

File: CHANGELOG.md (hunk 1 above)
Comment (yoavgeva) [reply CHANGELOG.md]: done

File: app/controllers/roles_controller.rb (hunk 2 above)
Comment (yoavgeva) [reply app/controllers/roles_controller.rb]: Fixed it.

**Pull request closed without merging**